### DISCUSSION TITLE
posts/hackintosh/ShinalonYAO7000-hackintosh

### DISCUSSION THREAD

**Discussion by giscus** [dreamhunter2333/dreamhunter-blog#14] [General]
# posts/hackintosh/ShinalonYAO7000-hackintosh

你指尖跃动的电光，是我此生不变的信仰

https://dreamhunter2333.com/posts/hackintosh/ShinalonYAO7000-hackintosh.html

**Comment by dreamhunter2333** [comment]
> 迁移数据
> "nick": "Anonymous"
> "createdAt": "2020-04-05T03:16:00.238Z"
> "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36"

那个EFI文件不行啊大佬

**Reply by dreamhunter2333** [reply]
> 迁移数据
> "createdAt": "2020-04-05T04:21:34.070Z",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36",

是我搞错了，EFI要在release里找。。。
